pkg/udev/mockdata.go

```diff
@@ -142,7 +143,9 @@ func OsDiskName() (string, string, error) {
 
 // getSyspathOfOsDisk returns syspath of os disk in success
 func getSyspathOfOsDisk(osDiskName string) (string, error) {
-	data, err := ioutil.ReadFile(filepath.Clean("/sys/class/block/" + osDiskName + "/dev"))
+	basePath := "/sys/class/block"
```

Comment:
```suggestion
	basePath := "/sys/class/block/"
```

just one more comment, we can make this consistent across the file.